packages/design-tokens/docs/StoryBoard.stories.tsx

```diff
@@ -251,20 +251,20 @@ const StoriesContainer = (props: {
       {shouldRender && (
         <Stack key={key}>
           {Object.entries(props.storyModule).map(([k, V]) => {
-            // Ignore exports that are Story Metadata (which we can detect by the presence of a "title" prop)
-            if ("title" in V) return null
-            const parameters = "story" in V ? V.story.parameters : V.parameters
+            // !This is now irrelevant as "title" prop is now string | undefined
+            // (outdated) Ignore exports that are Story Metadata (which we can detect by the presence of a "title" prop)
```

Comment:
Not sure if it's addressed in a later commit but we should clean up these outdated comments.

@dmisdm might be the best person to review for this file probably.